normandy/recipes/tests/api/v1/test_api.py

```diff
@@ -499,17 +505,11 @@ def test_filtering_by_enabled_fuzz(self, api_client):
                 "<%2fscript><svg%2fonload%3d'%2b%2f'%2f%2b"
             )
             res = api_client.get(url)
-            assert res.status_code == 400
-            assert res.data == {
-                'messages': [
-                    "'javascript:/*</script><svg/onload='+/'/+' "
-                    "value must be either True or False.",
```

Comment:
It would be nice, if it is easy. Most of the API works like this though: unrecognized fields or unexpected values in certain fields are just ignored, which isn't great. #1192 is related. Maybe some sort of larger filter schema checker is in order.

Reply:
I'm gonna leave this as is, if it's consistent with the behavior on the rest of the API. We should address this in v3.